sdks/java/io/amqp/pom.xml

```diff
@@ -39,6 +39,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
This refines the dependency because it is only used at this moment for testing purposes, if it wasn't the case the code won't compile.